plugins/sqs/README.md

```diff
@@ -0,0 +1,14 @@
+#[Description] (SQS plugin allows to push message using ECL code)
+   * First you have to put your aws accessKey in the credential file into .aws folder. This folder is located into home dir of hpcc process. "~/hpcc/.aws/" if it doesn't exist, you should create it.
```

Comment:
English:  "...credential file into the .aws folder. This folder is located in HPCC's home dir: "~/hpcc/.aws/". If it doesn't exist, you should create it".